### ISSUE TITLE
Missing DataTable download docs

### ISSUE THREAD

**Issue by chriddyp** [plotly/dash-docs#872]
would be good to follow up with https://community.plotly.com/t/allow-users-to-dowload-an-excel-in-a-click/9410/22 once this is written

**Comment by avats-dev** [comment]
I am thinking of adding "Export Options" on sidebar under datatable section in docs and then adding examples and commands for the same, but I don't have an idea of approaching it. How should I do it? @chriddyp

**Comment by chriddyp** [comment]
Good idea @avats-dev ! The community would be very grateful for your contribution.

To add a new chapter, create a new dictionary here: https://github.com/plotly/dash-docs/blob/dc754d5a2e20a13ae8e245e373f11fe5bc436a28/dash_docs/chapter_index.py#L469-L477. Each dictionary represents a chapter. These chapters will automatically appear in the sidebar.

Then, author the chapter in a new folder in https://github.com/plotly/dash-docs/tree/master/dash_docs/chapters/dash_datatable called "`export`". Create a file called `index.py` that has the `layout` variable assigned to it.

See the other chapters in the `dash_datatable` folder for inspiration.

**Comment by avats-dev** [comment]
Thanks @chriddyp for the help , I'm working on it. Will update soon!!

**Comment by adrienpacifico** [comment]
I think this issue can be closed : https://dash.plotly.com/dash-core-components/download (go to `Download Dataframe as Excel file`  section).

However, something bother me in the `.xlxs` extension that I refer to in #1151

**Comment by alexcjohnson** [comment]
@adrienpacifico this issue is still open, it's about documenting the clientside export feature of `dash_table.DataTable` - controlled by the props `export_format`, `export_columns`, and `export_headers`
